server/container/guice/cassandra-ldap-guice/src/main/java/org/apache/james/CassandraLdapJamesServerMain.java

```diff
@@ -0,0 +1,38 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james;
+
+import org.apache.james.data.LdapUsersRepositoryModule;
+import org.apache.james.modules.server.JMXServerModule;
+
+import com.google.inject.Module;
+import com.google.inject.util.Modules;
+
+public class CassandraLdapJamesServerMain {
+
+    public static void main(String[] args) throws Exception {
+        GuiceJamesServerImpl server = new GuiceJamesServerImpl()
+            .combineWith(CassandraJamesServerMain.cassandraServerModule, new JMXServerModule())
```

Comment:
To handle @aduprat comment, could you introduce cassandraLdapModule static field defined as : 

```
Modules.override(CassandraJamesServerMain.cassandraServerModule).with(new LdapUsersRepositoryModule());
```

Then you can simply use : 

```
GuiceJamesServerImpl server = new GuiceJamesServerImpl()
            .combineWith(cassandraLdapModule, new JMXServerModule());
```

Directly.